### PR TITLE
updated ui to allow for email searching

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: app/main/views/users.py [hunk 1]
```diff
@@ -18,17 +18,27 @@
...
     template = "view_users.html"
     users = None
+    teammembers = None
 
     email_address = request.args.get("email_address", None)
     if email_address:
-        users = data_api_client.get_user(email_address=request.args.get("email_address"))
+        users = data_api_client.get_user(email_address=email_address)
+
+    if users is None and email_address and "@" not in email_address:
```
Comment (matt-sm) [inline app/main/views/users.py]: why the check on `@`?

Comment (malmas18) [reply app/main/views/users.py]: to prevent calling of team members. that particular api method searches in the domain field where there is no @.

Comment (matt-sm) [reply app/main/views/users.py]: this may not be obvious to the user - may need to tell them thats how it works

Comment (malmas18) [reply app/main/views/users.py]: i'll add a comment to to the code and update the example of the field

**Pull request merged**